parser/parser.py

```diff
@@ -10,6 +22,12 @@ def get_version():
         ["git", "describe", "--tags", "--abbrev=0"], capture_output=True
```

Comment:
deze geeft bij mij: AttributeError: 'NoneType' object has no attribute 'startswith'
mogelijk te maken met dit : 
https://stackoverflow.com/questions/6445148/git-fatalno-tags-can-describe-sha1-number

Reply:
Dat komt omdat je geen git fetch hebt gedaan.

Reply:
Wanneer je een git fetch doet krijg je alle tags binnen (de laatste is 0.0.6). Iedereen die de repo gebruikt krijgt deze ook. Maar ik zal er nog een check omheen zetten.